modules/tinymce/src/plugins/table/main/ts/ui/MenuItems.ts

```diff
@@ -171,7 +170,7 @@ const addMenuItems = (editor: Editor, selections: Selections, selectionTargets:
     editor.ui.registry.addMenuItem('inserttable', {
       text: 'Table',
       icon: 'table',
-      onAction: cmd('mceInsertTable')
+      onAction: cmd('mceInsertTableDialog')
```

Comment:
Is this going to require a changelog entry?

Reply:
Yep, I need to work out what changelog entries need to be added. This will be one of them.

Reply:
Changelog entiy has been added.